src/workflow/ArcanistPatchWorkflow.php

```diff
@@ -511,17 +511,18 @@ public function run() {
           "<bg:blue>** %s **</bg> %s\n",
           pht('INFO'),
           pht('Base commit is not in local repository; trying to fetch.'));
-        $repository_api->execManualLocal('fetch --quiet --all');
+        // UBER CODE
```

Comment:
Are you sure this part is even used by submitqueue, afaik you rely on custom patching from ref and you probably won't even go here?